share/hooks/raft/vip.sh

```diff
@@ -17,7 +17,7 @@ fi
 case $ACTION in
 leader)
     sudo ip address add $IP dev $INTERFACE
-    arping -c 5 -U -I $INTERFACE ${IP%%/*}
+    sudo arping -c 5 -U -I $INTERFACE ${IP%%/*}
```

Comment:
Sudo shouldn't be needed as the arping binary (should) have the file capabilities set:

```
$ getcap $(which arping)
/usr/sbin/arping = cap_net_raw+p
```

But, it can be better to have sudo there.

Just curious -- what platform, filesystem,  (SELinux/AppArmor?) do you use and can you copy&paste the executed command and error output? Not necessary, thank you.